commands/host.vegman

```diff
@@ -354,6 +354,20 @@ function cmd_config_default {
   config_default_${action} ${yes}
 }
 
+# @sudo cmd_config_bifurcation admin,operator,user
+# @doc cmd_config_bifurcation
+# Host bifurcation
+function cmd_config_bifurcation {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_bifurcation_show admin,operator,user
+# @doc cmd_config_bifurcation_show
+# Show host bifurcation
+function cmd_config_bifurcation_show {
```

Comment:
Hey, what are you doing here? What is the idea of the change? I feel it is not what you expect.

Reply:
My task says to add `host config bifurcation show` command to the CLI using the host-pcie-cfg command that you (iirc) implemented.